.github/workflows/publish-npm-package.yml

```
@@ -1,52 +1,79 @@
-# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
-# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
-
-name: Publish NPM Package
-
-on:
-  release:
-    types: [created]
-  workflow_dispatch:
-    inputs:
-      release-version:
-        required: true
-
-jobs:
-  publish-gpr:
-    runs-on: ubuntu-latest
-    permissions:
-      contents: read
-      packages: write
-    steps:
-      - name: Checkout
-        uses: actions/checkout@master
-        with:
-          ref: "master"
-
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 16
-          registry-url: "https://registry.npmjs.org"
-          scope: "@layer5"
-
-      - name: Install Dependencies
-        run: npm ci
-
-      - name: Run npm release
-        run: npm run publish-ts-lib ${{ github.event.inputs.release-version || github.event.release.tag_name }}
-        if: github.event_name == 'workflow_dispatch' || github.event_name == 'release'
-
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-
-      - name: Pull changes from remote
-        run: git pull origin master
-
-      - name: Commit and push version change
-        uses: stefanzweifel/git-auto-commit-action@v4
-        with:
-          commit_user_name: l5io
-          commit_user_email: ci@layer5.io
-          commit_author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
-          commit_options: "--signoff"
-          commit_message: "[Release]: Bump version to ${{ github.event.inputs.release-version || github.event.release.tag_name }}"
\ No newline at end of file
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Publish NPM Package
+
+on:
+  workflow_dispatch:
```

```suggestion
  release:
    types: [published]
  workflow_dispatch:
```
Adding trigger on release.